components/gc-page-success/gc-page-success-new-en.html

```diff
@@ -0,0 +1,118 @@
+						<button type="button" class="btn btn-primary mrgn-lft-sm" data-wb-doaction="[
+						{&quot;action&quot;:&quot;removeClass&quot;,&quot;source&quot;:&quot;.gc-pg-hlpfl-no&quot;,&quot;class&quot;:&quot;nojs-show&quot;},
+						{&quot;action&quot;:&quot;addClass&quot;,&quot;source&quot;:&quot;.gc-pg-hlpfl-btn&quot;,&quot;class&quot;:&quot;hide&quot;}
+						]" data-gc-analytics-wtph-no>No</button>
+					</div>
+				</fieldset>
+				<p id="why-note" class="nojs-show mrgn-tp-lg mrgn-bttm-md"><strong>If not, tell us why below:</strong></p><!--NOTE: For noscript and basic HTML modes.-->
+				<div class="gc-pg-hlpfl-no nojs-show">
+					<p class="nojs-hide wb-inv"><strong>Tell us why below:</strong></p><!--NOTE: For JS mode after the transition (to instruct screen readers about what to do next).-->
+					<div aria-live="off">
+						<fieldset class="chkbxrdio-grp">
+							<legend><span class="field-name">What was wrong?</span></legend>
+							<div class="radio">
+								<label for="problem1">
+									<input name="problem" id="problem1" type="radio" value="I can't find the information" data-gc-analytics-wtph-value="I can't find the information-Je ne peux pas trouver l'information" data-gc-analytics-collect="notPrivate"> I can't <strong>find</strong> the information
+								</label>
+							</div>
+							<div class="radio">
+								<label for="problem2">
+									<input name="problem" id="problem2" type="radio" value="The information is hard to understand" data-gc-analytics-wtph-value="The information is hard to understand-L'information est difficile à comprendre" data-gc-analytics-collect="notPrivate"> The information is hard to <strong>understand</strong>
+								</label>
+							</div>
+							<div class="radio">
+								<label for="problem3">
+									<input name="problem" id="problem3" type="radio" value="There was an error / something didn't work" data-gc-analytics-wtph-value="There was an error or something didn't work-Il y avait une erreur ou quelque chose ne fonctionnait pas" data-gc-analytics-collect="notPrivate"> There was an error or something <strong>didn't work</strong>
+								</label>
+							</div>
+							<div class="radio">
+								<label for="problem4">
+									<input name="problem" id="problem4" type="radio" value="Other reason" data-gc-analytics-wtph-value="Other reason-Autre raison" data-gc-analytics-collect="notPrivate"> Other reason
+								</label>
+							</div>
+						</fieldset>
+						<div class="form-group">
+							<label for="problem6" class="mrgn-tp-sm mrgn-bttm-0"><span class="field-name">Please provide more details (maximum 300 characters)</span></label>
+							<textarea id="problem6" aria-describedby="problem6-note" name="details" class="form-control full-width" maxlength="300"></textarea>
+							<p id="problem6-note"><small>You will not receive a reply. Telephone numbers and email addresses will be removed.</small></p>
```

Comment:
```suggestion
							<label for="problem6" class="mrgn-tp-sm mrgn-bttm-0"><span class="field-name">Please provide more details </span></label>
							<p id="problem6-note"><small><strong>You will not receive a reply. Telephone numbers and email addresses will be removed.</strong><br>Maximum 300 characters</small></p>
							<textarea id="problem6" aria-describedby="problem6-note" name="details" class="form-control full-width" maxlength="300"></textarea>
```
More up to date layout

Reply:
This change was intentional on my part.

IMO the character limit is must-need information for anyone filling out the details field. Users are likely to care more about the short character limit than a side note about what'll happen after submitting the form. So I figured it'd be wise to place the character limit in the label itself to make it prominent.

FWIW the paragraph is currently associated to the field via an ``aria-describedby`` attribute. If the character limit had been kept at the end of the paragraph, it would've been harder for sighted users to spot. Plus screen readers would've announced it at the very end - after the label, field type and side note. So important info that's directly related to the field would've be treated as if it was the least important part of the secondary information.

Thoughts? I could go with your change if you insist, but IMO it'd represent a downgrade vs what I currently have.

Reply:
You've convinced me at Screen readers.